doc/installation.rst

```diff
@@ -1,9 +1,30 @@
 ##########################################
-Installation
+Getting started
 ##########################################
 
-Section A
-=========
+Installation
+============
+
+There are three possibilities to install atlite:
+
+* conda::
+
+    conda install -c conda-forge atlite
+
+
+* pypsi::
+
+    pip install atlite
+
+* git::
+
+    pip install git+https://github.com/PyPSA/atlite/
+
+Get the Datasets
+================
+
+Download one of the weather datasets listed above.
 
-Section B
-=========
+ERA5 is downloaded automatically on-demand after the `ECMWF ADS API <https://cds.climate.copernicus.eu/api-how-to>`_
+(European Centre for Medium-Range Weather Forecasts Climate Data Store
+Application Program Interface) client is properly installed.
```

Comment:
"See separate, linked installation guide for details, especially for correctly setting up your CDS API key."